src/pages/confs/2017_jsconf_eu/a_cartoon_intro_to_webassembly.md

```diff
@@ -50,5 +50,6 @@ Nice talk about compilers and webassembly. Very easy to understand, but not very
 
 Related Links | Rating
 --- | ---
+[Video](https://www.youtube.com/watch?v=HktWin_LPf4)
```

Comment:
Videos are added to the metadata `video` field. This one, for example, already has it.